src/commands/users/kaokaiToday.ts

```diff
@@ -0,0 +1,123 @@
+import axios from 'axios'
+import cheerio from 'cheerio'
+
+import { CommandHandlerConfig } from '../../types/CommandHandlerConfig.js'
+import { CommandInteraction } from '../../../node_modules/discord.js/typings/index.js'
+
+//Fetch HTML Func
+async function fetchHTML(url: string): Promise<string> {
+  try {
+    const response = await axios.get(url)
+    return response.data
+  } catch (error) {
+    console.error('Error fetching HTML:', error)
+    throw error
+  }
+}
+
+export default {
+  data: {
+    name: 'kaokai-today',
+    description: 'Show 10 forum topics sorted according to your preferences.',
+    options: [
+      {
+        name: 'latest',
+        description: 'ข่าวสารก้าวไกลทั้งหมดอัพเดตล่าสุด',
+        type: 1,
+      },
+      {
+        name: 'highlight',
+        description: 'ไฮไลต์ข่าวสารก้าวไกล',
+        type: 1,
+      },
```

Comment:
Description should be English.
```ts
description: 'Latest and up to date news from Move Forward Party',
```
then you can add Thai translation using `descriptionLocalizations`.
```ts
descriptionLocalizations: {
  th: 'ข่าวสารก้าวไกลทั้งหมดอัพเดตล่าสุด',
},
```

```ts
{
  name: 'latest', 
  description: 'Latest and up to date news from Move Forward Party',
  descriptionLocalizations: {
    th: 'ข่าวสารก้าวไกลทั้งหมดอัพเดตล่าสุด',
  },
  type: 1,
},
{
  name: 'highlight',
  description: 'Highlight news from Move Forward Party',
  descriptionLocalizations: {
    th: 'ไฮไลต์ข่าวสารก้าวไกล',
  },
  type: 1,
},
```

Reply:
Edited!